ansible/templates/golden_config_db_t2.j2

```diff
@@ -0,0 +1,30 @@
+{
+{%- set asic_cnt = asic_cnt|int -%}
+{% if asic_cnt > 1 %}
+  "localhost": {
+    "MACSEC_PROFILE": {}
+  },
+{% for ns in range(asic_cnt) %}
+  "asic{{ns}}": {
```

Comment:
is this the way the config will get generated for single asic system ?

Reply:
It will generate an empty table "MACSEC_PROFILE": {}

Reply:
I think this is ok now, but there are some single asic Linecards which support macsec as well. Can we enhance this in the next PR?

Reply:
@judyjoseph : you will create test gap for this ?